download/download.go

```diff
@@ -311,6 +311,23 @@ func (d *Downloader) download(ctx context.Context, m metrics.Metrics) (*download
 			if d.sizeLimitBytes != nil {
 				reader = reader.WithSizeLimitBytes(*d.sizeLimitBytes)
 			}
+
+			if d.logger.GetLevel() >= logging.Debug {
+				expectedBundleContentType := []string{
+					"application/gzip",
+					"application/octet-stream",
+				}
```

Comment:
`application/vnd.openpolicyagent.bundles` is used as the content type to indicate the server supports long polling.

Reply:
Ah, good input! I'll have that added.